MekHQ/src/mekhq/gui/dialog/CampaignOptionsDialog.java

```diff
@@ -1516,6 +1517,11 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         gridBagConstraints.gridy = 21;
         panPersonnel.add(chkUseTransfers, gridBagConstraints);
 
+        chkUseTimeInService = new JCheckBox("Use Time In Service (Unofficial)"); // NOI18N
```

Comment:
Is this really an unofficial option?  It seems like this is just additional tracking.  I might change this to "Track Time in Service".

Reply:
It's not rules related, but could drop the unofficial I guess.